README.md

```diff
@@ -55,26 +80,83 @@ cosign verify --key cosign.pub ghcr.io/avisi-cloud/structurizr-site-generatr:<ve
 
 Replace `<version>` with the version you want to verify.
 
-## Example usage
+## Usage
```

Comment:
```suggestion
## Usage

Note: In the examples below, the `<version>` suffix should be replaced with your chosen version number. For example
`docker pull ghcr.io/avisi-cloud/structurizr-site-generatr:1.0.9`.
```

Reply:
I agree that starting with a note is less than ideal. I also agree that adding a `latest` tag is a better solution. So let's keep it like this for now.